tests/ui/consts/issue-17718-const-bad-values.rs

```diff
@@ -7,8 +7,4 @@
 const C1: &'static mut [usize] = &mut [];
 //~^ ERROR: mutable references are not allowed
 
-static mut S: i32 = 3;
-const C2: &'static mut i32 = unsafe { &mut S };
-//~^ ERROR: reference to mutable memory
-
```

Comment:
Why was this removed? It doesn't seem like its behavior was changed by this PR (it still generates the same E0080 error). Is it covered in some other test?

Reply:
Uh, I don't remember...
The error message does change, so it is unclear whether this even still tests the right thing. :shrug: But I can add the test back with the new error, sure.